.github/workflows/docs-ci.yml

```diff
@@ -4,9 +4,45 @@ on:
   pull_request:
     branches: [ main ]
   push:
-    branches: [ 'main', 'release/v*' ]
+    branches: [ '**', 'release/v*' ]
+
+env:
+  version-path: ${{ startsWith(github.ref, 'refs/heads/release/v') && github.ref_name || 'release/v0.30.0' }}
```

Comment:
I'm assuming this `|| 'release/v0.30.0'` is just temporary?

Reply:
Yes.  For testing.  But I should have changed it back for re-review.